packages/gateway/src/workflow/index.ts

```diff
@@ -281,6 +280,29 @@ export async function rejectRegistration(
   return taskEntry
 }
 
+export async function upsertRegistrationIdentifier(
+  id: string,
+  authHeader: IAuthHeader,
+  details: {
+    registrationNumber: string
```

Comment:
@tahmidrahman-dsi do we need the registrationNumber at all here? can we just push the identifiers with a registration id

Reply:
If it helps, this can even just be a 
```
      mutation upsertRegistrationIdentifier(
        $eventId: ID!
        $identifierType: String!
        $identifierValue: String!
      ) {
```
as you can call the endpoint multiple times if you need to add multiple identifiers